services/director-v2/src/simcore_service_director_v2/utils/rabbitmq.py

```diff
@@ -106,11 +111,11 @@ async def publish_service_resource_tracking_started(  # pylint: disable=too-many
         project_name=project_name,
         node_id=node_id,
         node_name=node_name,
-        parent_project_id=project_id,  # <-- SAN please modify
-        root_parent_project_id=project_id,  # <-- SAN please modify
-        root_parent_project_name=project_name,  # <-- SAN please modify
-        parent_node_id=node_id,  # <-- SAN please modify
-        root_parent_node_id=node_id,  # <-- SAN please modify
+        parent_project_id=parent_project_id or project_id,
+        root_parent_project_id=root_parent_project_id or project_id,
+        root_parent_project_name=root_parent_project_name or project_name,
+        parent_node_id=parent_node_id or node_id,
+        root_parent_node_id=root_parent_node_id or node_id,
```

Comment:
Super thanks! And yes if you think this logic should be moved to RUT I can move it there afterwards.

Reply:
discussed. I let it be for now.